### ISSUE TITLE
Staging Review finding: Form flow focus management doesn't match other VA.gov forms

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99402]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

This ticket was part of a merged Staging Review with MHV Medical Records [Download Records](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92340), [Flow/Images](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95081) and [Settings](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95082)

## Product Information
**Team:** vfs-mhv-medical-records
**Product:** MHV Medical Records
**Feature:** Phase 1 Radiology Flow/Images, Download Records, and Settings

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that doesn't match other content on VA.gov.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-is-similar-to-other-content-on-vagov)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

As a user moves through the Blue Button form flow, keyboard focus is sent to the H1 when progressing to the next page of the flow. For screen reader users, the element receiving keyboard focus is the first thing announced. Since the H1 is static throughout the form flow, screen reader users hear the same thing on each page: "Select records and download report."

This doesn't provide screen reader users with useful information about where they are in the flow, and doesn't match the behavior of other forms on VA.gov. Other forms send keyboard focus to the step indicator for each page, or alternatively to the first unique heading on each page.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to reproduce:
1. Navigate to the Blue Button form flow.
2. Activate a screen reader.
3. Navigate through the form flow.
4. Note that on each page transition, "Select records and download report" receives focus and is announced.

</details>

## Recommended action

I had thought the page transition focus management was included as part of the forms library, but if it's not then make sure you're matching the behavior of the [mock form patterns](https://staging.va.gov/mock-form-patterns/introduction) and reach out to the forms team for implementation guidance.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by lichellebain** [comment]
https://jira.devops.va.gov/browse/MHV-65357

**Comment by bryan-riley-va** [comment]
Resolved/Closed 2/19/25
